ui/widgets/nuxeo-select.js

```diff
@@ -197,6 +199,31 @@ import { IronResizableBehavior } from '@polymer/iron-resizable-behavior/iron-res
           value: null,
         },
 
+        /**
+         * The orientation against which to align the menu dropdown
+         * horizontally relative to the dropdown trigger.
+         */
+        horizontalAlign: {
+          type: String,
+          value: 'left',
+        },
+
+        /**
+         * The orientation against which to align the menu dropdown
+         * vertically relative to the dropdown trigger.
+         */
+        verticalAlign: {
+          type: String,
+          value: 'top',
+        },
+
+        /**
+         * Dynamically align dropdown menu.
+         */
+        dynamicAlign: {
```

Comment:
what is dynamic alignment?

Reply:
It is a property from `paper-dropdown-menu` that calculates the menu position. (see [here](https://github.com/PolymerElements/paper-dropdown-menu/blob/master/paper-dropdown-menu.js#L213-L219))

It will, basically, ensure that it fits the screen. Example:
![Screenshot 2020-09-03 at 15 06 10](https://user-images.githubusercontent.com/16035473/92125572-15670700-edf7-11ea-9cad-a9a09bec7117.png)

vs

![Screenshot 2020-09-03 at 15 07 33](https://user-images.githubusercontent.com/16035473/92125719-3a5b7a00-edf7-11ea-8553-1068b8f39877.png)